test/util/request_processor.dart

```diff
@@ -0,0 +1,68 @@
+import 'dart:convert';
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+
+import 'request_processor.mocks.dart';
+
+@GenerateMocks([RequestProcessor])
+class RequestProcessor {
+  String process(String url, {String? method, Map<String, dynamic>? body}) {
+    throw UnimplementedError();
```

Comment:
this now throws an error, just to mak sure that we know when we forgot to mock one of the HTTP calls.